Source/WebCore/Modules/permissions/PermissionStatus.cpp

```diff
@@ -30,49 +30,84 @@
 #include "Document.h"
 #include "DocumentInlines.h"
 #include "EventNames.h"
+#include "MainThreadPermissionObserver.h"
 #include "PermissionController.h"
+#include "PermissionState.h"
+#include "Permissions.h"
+#include "ScriptExecutionContext.h"
 #include "SecurityOrigin.h"
+#include "WorkerGlobalScope.h"
+#include "WorkerLoaderProxy.h"
+#include "WorkerThread.h"
+#include <memory>
```

Comment:
Why is this needed?

Reply:
Used for unique_ptr. But I suppose this can be removed.